packages/suite/src/support/messages.ts

```diff
@@ -4653,6 +4653,10 @@ export default defineMessages({
         defaultMessage:
             'In all cases, we strongly recommend <button>creating a wallet backup in Settings</button>. With a wallet backup, your coins are safe and secure even if your Trezor is lost, damaged, or stolen.',
     },
+    TR_CHECK_BACKUP: {
```

Comment:
Let's just use `TR_CHECK_RECOVERY_SEED`, I don't think we need a new string for this.